rules/manager_test.go

```diff
@@ -789,18 +789,27 @@ type ruleGroupsTest struct {
 
 // ruleGroupTest forms a testing struct for running tests over rules.
 type ruleGroupTest struct {
-	Name     string         `yaml:"name"`
+	Name     yaml.Node      `yaml:"name"`
 	Interval model.Duration `yaml:"interval,omitempty"`
-	Rules    []rulefmt.Rule `yaml:"rules"`
+	Rules    []rule         `yaml:"rules"`
+}
+
+type rule struct {
```

Comment:
Why do you need these copies of the existing structs?

Reply:
Addressed this.